src/components/dialogs/csv-import-filter-creation-dialog.js

```diff
@@ -127,14 +128,23 @@ const CsvImportFilterCreationDialog = ({
     };
 
     const handleOpenCSVConfirmationDataDialog = () => {
-        const res = tableValues.filter(
-            (d) => !Object.values(d).every((v) => v == null)
-        ).length;
-        if (res === 0) {
+        // We check if there are values in the table
+        const isValuesInTable = tableValues.some(
```

Comment:
rename for isTableFIlled ? areSomeValuesPresent ?

Reply:
isTableFilled won't work because it implies the table has no empty cell (which this code do not test), and I'd rather keep the "isXXX" convention for boolean values.